GameView.m

```diff
@@ -99,9 +99,11 @@ - (void) drawRect: (NSRect) rect
         else
             operation = NSCompositeSourceOver;
                 
-        [image compositeToPoint: origin
-                      operation: operation
-                       fraction: card == nil? 0.5 : 1.0 ];
+        //[image compositeToPoint: origin
+        //             operation: operation
+        //               fraction: card == nil? 0.5 : 1.0 ];
+        [ image drawAtPoint: origin fromRect: frame operation: operation
```

Comment:
It’s probably better to send `fromRect:` an `NSZeroRect`. From the documentation:

> If you pass in `NSZeroRect`, the entire image is drawn.